packages/pools/src/types.ts

```diff
@@ -48,8 +49,14 @@ export function makeStaticPoolFromRaw(
     );
   }
   if (rawPool["@type"] === COSMWASM_POOL_TYPE) {
-    // currently only support transmuter pools
-    return new TransmuterPool(rawPool as CosmwasmPoolRaw);
+    rawPool = rawPool as CosmwasmPoolRaw;
+
+    // differentiate cosmoswasm pools by code id - 5005 would be for testnet
+    if (rawPool.code_id === "5005") {
```

Comment:
This should be done in the web/config like transmuter code ids:

https://github.com/osmosis-labs/osmosis-frontend/blob/stage/packages/web/config/feature-flag.ts#L50

Reply:
See latest comment with link to PR containing the branch to target instead of stage.